src/baitapdateclass/Main.java

```diff
@@ -0,0 +1,68 @@
+package baitapdateclass;
+
+import java.util.Scanner;
+
+public class Main {
+
+	public static void main(String[] args) {
+		int date, month, year;
+		Scanner scanner = new Scanner(System.in);
+		System.out.println("Nhập ngày: ");
+		date = scanner.nextInt();
+		System.out.println("Nhập tháng: ");
+		month = scanner.nextInt();
+		System.out.println("Nhập năm: ");
+		year = scanner.nextInt();
+
+		switch (month) {
+		case 1:
+		case 3:
+		case 5:
+		case 7:
+		case 8:
+		case 10:
+		case 12:
+			if (date > 0 && date <= 31) {
+				System.out.println("Ngày tháng năm hợp lệ");
+				break;
+			} else {
+				System.out.println("Ngày tháng năm không hợp lệ");
+				break;
+			}
+
+		case 4:
+		case 6:
+		case 9:
+		case 11:
+			if (date <= 30 && date > 0) {
+				System.out.println("Ngày tháng năm hợp lệ");
+				break;
+			} else {
+				System.out.println("Ngày tháng năm không hợp lệ");
+				break;
+			}
+
+		case 2:
+			if (year % 4 == 0 || year % 100 == 0 || year % 400 == 0 && date <= 29 && date > 0) {
+				System.out.println("Đây là năm nhuận và ngày tháng năm hợp lệ");
+				break;
+			} else if (date <= 28 && date > 0) {
+				System.out.println("Ngày tháng năm hợp lệ");
+				break;
+			} else {
+				System.out.println("Ngày tháng năm không hợp lệ");
+				break;
+			}
```

Comment:
```suggestion
			if (year % 4 == 0 || year % 100 == 0 || year % 400 == 0 && date <= 29 && date > 0) {
				System.out.println("Đây là năm nhuận và ngày tháng năm hợp lệ");
				break;
			} 
			if (date <= 28 && date > 0) {
				System.out.println("Ngày tháng năm hợp lệ");
				break;
			}
			System.out.println("Ngày tháng năm không hợp lệ");
			break;
```

ngoài ra, chi tháy em check ngày, tháng, năm  < 0 ở tỏng từng if, sao em ko check trc là ngày, tháng, năm phải >0 và ngày < 32, tháng nhỏ hơn 12 thì mới cho vào switch case nhỉ. như vậy nhanh gọn lẹ hơn nhìu bé

ko clear comment thì hỏi lại chị